app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,13 @@
+class DashboardController < ApplicationController
+  def index
+    @total_learned_kanji = total_learned_kanji
+    @total_kanji = Kanji.count
```

Comment:
Depends what we're trying to do, but this number is based on all kanji in our dictionary.
I think it would also be nice to have a display of 
Learned_Kanji / Captured_Kanji